packages/usa-alert/src/usa-alert.stories.js

```diff
@@ -1,6 +1,11 @@
 import Component from "./usa-alert.twig";
 import TestComponent from "./test/test-patterns/test-usa-alert-lists.twig";
 import ComparisonComponent from "./test/test-patterns/test-alert-comparison.twig";
+import onPageComponent from "./test/test-patterns/test-alert-in-template.twig";
+import BannerContent from "../../usa-banner/src/content/usa-banner.json";
+import FooterContent from "../../usa-footer/src/usa-footer.json";
+import SiteAlertContent from "../../usa-site-alert/src/content/usa-site-alert~info.json";
+import SidenavContent from "../../usa-sidenav/src/usa-sidenav.json";
```

Comment:
**quibble (non-blocking):** I'm curious if there's a way to collect this data in the twig template without defining each line how we were before. Since this is just for the test story, I'd be interested in excluding it from the `stories.js` file.

That said, I toyed around with importing the JSON directly to the template with no luck.

```twig
{% set bannerContent %}
  {% include "@components/usa-banner/src/content/usa-banner.json" %}
{% endset %}

{% include '@components/usa-banner/src/usa-banner.twig' with bannerContent %}
```

Do you know of anyway we could accomplish this?